recipe/meta.yaml

```diff
@@ -27,42 +27,100 @@ source:
     - nvcc.profile.patch.win  # [win]
 
 build:
-  number: 1
+  number: 2
   skip: true  # [osx or win]
 
-requirements:
-  build:
-    - {{ compiler("c") }}
-    - {{ compiler("cxx") }}
-    - arm-variant * {{ arm_variant_type }}  # [aarch64]
-    - sysroot_{{ target_platform }} 2.17    # [linux]
-  host:
-    - cuda-version {{ cuda_version }}
-  run:
-    - {{ pin_compatible("cuda-version", max_pin="x.x") }}
-    - cuda-cudart-dev
-  run_constrained:
-    - gcc_impl_{{ target_platform }} >=6,<13  # [linux]
-    - arm-variant * {{ arm_variant_type }}  # [aarch64]
-    - cmake >=3.20
+outputs:
+  - name: cuda-nvcc
+    files:
+      - bin                                               # [linux]
+      - targets/{{ target_name }}/bin                     # [linux]
+      - targets/{{ target_name }}/nvvm                    # [linux]
+      - lib/libnvptxcompiler_static.a                     # [linux]
+      - nvvm                                              # [linux]
+      - etc/conda/activate.d/~cuda-nvcc*_activate.sh      # [linux]
+      - etc/conda/deactivate.d/~cuda-nvcc*_deactivate.sh  # [linux]
+      - Library\nvvm                                      # [win]
+    requirements:
+      build:
+        - {{ compiler("c") }}
+        - {{ compiler("cxx") }}
+        - arm-variant * {{ arm_variant_type }}  # [aarch64]
+        - sysroot_{{ target_platform }} 2.17    # [linux]
+      host:
+        - cuda-version {{ cuda_version }}
+      run:
+        - {{ pin_compatible("cuda-version", max_pin="x.x") }}
+        - {{ pin_subpackage("cuda-nvcc_" + target_platform, exact=True) }}
+        - cuda-cudart-dev
+      run_constrained:
+        - gcc_impl_{{ target_platform }} >=6,<13  # [linux]
+        - arm-variant * {{ arm_variant_type }}    # [aarch64]
```

Comment:
Looks like we lost the cmake constraint added in Rob's PR this morning.

Reply:
Oh I see it looks like that's in the target output instead. That seems fine to me too.

Reply:
Yeah since `cuda-nvcc` depends on `cuda-nvcc_{{ target_platform }}`, which has the `cmake` constraint. `cuda-nvcc` gets the `cmake` constraint for free